ReactSkia/core_modules/RSkInputEventManager.cpp

```diff
@@ -17,15 +21,31 @@ RSkInputEventManager::RSkInputEventManager(){
                                                               std::placeholders::_2);
   eventId_ = NotificationCenter::defaultCenter().addListener("onHWKeyEvent", handler);
   spatialNavigator_ =  SpatialNavigator::RSkSpatialNavigator::sharedSpatialNavigator();
+  keyRepeat=false;
+  previousKeyType=RNS_KEY_UnKnown;
 }
 
 void RSkInputEventManager::keyHandler(rnsKey eventKeyType, rnsKeyAction eventKeyAction){
   bool stopPropagate = false;
-  if(eventKeyAction != RNS_KEY_Press)
-    return;
+  RNS_LOG_DEBUG("[RSkInputEventManager] [keyHandler] Key Repeat" << keyRepeat<<"  eventKeyType  " <<eventKeyType << " previousKeyType " <<previousKeyType <<"  eventKeyAction  " << eventKeyAction);
+  
+  if(previousKeyType == eventKeyType  && eventKeyAction == previousKeyAction && !keyRepeat){
```

Comment:
as discussed, need to check possibility to improve this logic here to avoid multiple checks to decide keyRepeat